index.html

```diff
@@ -37,7 +37,7 @@
   // TODO: Sort the names in alphabetical order
   // TODO: remove every name starting with the letter J
 
-  document.getElementById('result').innerHTML = names;
+  document.getElementById('result').innerHTML = names.sort();
```

Comment:
If you reload the page are the names sorted?

Reply:
sorted by first name, is that not what we need?